pandas/tests/indexing/test_categorical.py

```diff
@@ -627,15 +627,79 @@ def test_reindexing(self):
                       lambda: self.df2.reindex(['a'], limit=2))
 
     def test_loc_slice(self):
-        # slicing
-        # not implemented ATM
-        # GH9748
+        # Raises KeyError since the left slice 'a' is not unique
+        pytest.raises(KeyError, lambda: self.df.loc["a":"b"])
+        result = self.df.loc["b":"c"]
```

Comment:
this tests is the same as the 3rd case? if so do we need both?  (or if not can you move them together and comment)